queens/drivers/jobscript_driver.py

```diff
@@ -143,18 +148,53 @@ def create_input_templates_dict(input_templates):
         }
         return input_templates_dict
 
+    @staticmethod
+    def get_read_in_jobscript_template(jobscript_template):
+        """Get the jobscript template contents.
+
+        If the provided jobscript template is a Path or a string of a
+        path and a valid file, the corresponding file is read.
+
+        Args:
+            jobscript_template (str, Path): Path to jobscript template or read-in jobscript
+                                            template.
+
+        Returns:
+            str: Read-in jobscript template
+        """
+        if isinstance(jobscript_template, str):
+            # Catch an exception due to a long string
+            try:
+                if Path(jobscript_template).is_file():
+                    jobscript_template = read_file(jobscript_template)
+            except OSError:
+                # We assume that the string already holds the jobscript template contents
+                pass
```

Comment:
I find that adding `pass` in exceptions might lead to problems down the line. Should we some debug logging `using the jobscript template as string` and possibly even logging the contents